User/Entities/UserEntity.cs

```diff
@@ -0,0 +1,12 @@
+namespace User.Entities;
+
+public class UserEntity
+{
+    public int Id { get; set; }
+    public string Name { get; set; } = null!;
+    public string Email { get; set; } = null!;
+    public string Number { get; set; } = null!;
+    public string? Description { get; set; }
+    public string Password { get; set; } = null!;
+    public UserRoleEnum Role { get; set; }
```

Comment:
I don't really know do we really need this `enum`?
I've used it in my coursework but it was actually a pain in the as. cuz I needed to transfer user roles as `0` and `1` through the network
I wish I'd used just a default `String` type

Reply:
Removed Rome enum. Replaced it with entity which is stored in database. In future will be endpoint to get full list of roles.